Modelica/Electrical/PowerConverters/Examples/DCDC/ExampleTemplates/ChopperBuckBoost.mo

```diff
@@ -0,0 +1,120 @@
+    annotation (Placement(transformation(extent={{50,40},{30,60}})));
+  Modelica.Electrical.PowerConverters.DCDC.Control.SignalPWM pwm(f=fS) annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={20,-40})));
+  Modelica.Electrical.Analog.Sensors.CurrentSensor currentSensor
+    annotation (Placement(transformation(extent={{-60,-30},{-80,-10}})));
+equation
+  connect(constantVoltageLV.p, resistorLV.p)
+    annotation (Line(points={{-90,10},{-90,20},{-80,20}}, color={0,0,255}));
+  connect(constantVoltageLV.n, ground.p)
+    annotation (Line(points={{-90,-10},{-90,-20}},           color={0,0,255}));
+  connect(resistorHV.p, constantVoltageHV.p)
+    annotation (Line(points={{80,20},{90,20},{90,10}}, color={0,0,255}));
+  connect(vLV.y, mean_vLV.u)
+    annotation (Line(points={{-59,80},{-52,80}}, color={0,0,127}));
+  connect(iLV.y,mean_iLV. u)
+    annotation (Line(points={{-59,50},{-52,50}}, color={0,0,127}));
+  connect(mean_iHV.u, iHV.y)
+    annotation (Line(points={{52,50},{59,50}}, color={0,0,127}));
+  connect(vHV.y, mean_vHV.u)
+    annotation (Line(points={{59,80},{52,80}}, color={0,0,127}));
+  connect(capacitorHV.n, dcdc.dc_n2) annotation (Line(points={{50,-10},{50,-20},
+          {40,-20},{40,-4},{30,-4}}, color={0,0,255}));
+  connect(dcdc.dc_p2, resistorHV.n)
+    annotation (Line(points={{30,8},{40,8},{40,20},{60,20}}, color={0,0,255}));
+  connect(capacitorHV.p, resistorHV.n)
+    annotation (Line(points={{50,10},{50,20},{60,20}},
+                                               color={0,0,255}));
+  connect(capacitorHV.n, constantVoltageHV.n) annotation (Line(points={{50,-10},
+          {50,-20},{90,-20},{90,-10}}, color={0,0,255}));
+  connect(resistorLV.n, resistor.p)
+    annotation (Line(points={{-60,20},{-50,20}}, color={0,0,255}));
+  connect(resistor.n, inductor.p)
+    annotation (Line(points={{-30,20},{-20,20}}, color={0,0,255}));
+  connect(dcdc.dc_p1, inductor.n)
+    annotation (Line(points={{10,8},{0,8},{0,20}},          color={0,0,255}));
+  connect(dcdc.fire_p, pwm.fire)
+    annotation (Line(points={{14,-10},{14,-29}}, color={255,0,255}));
+  connect(dcdc.fire_n, pwm.notFire)
+    annotation (Line(points={{26,-10},{26,-29}}, color={255,0,255}));
+  connect(resistor.p, capacitorLV.p)
+    annotation (Line(points={{-50,20},{-50,10}}, color={0,0,255}));
+  connect(ground.p, currentSensor.n)
+    annotation (Line(points={{-90,-20},{-80,-20}}, color={0,0,255}));
+  connect(currentSensor.p, capacitorLV.n)
+    annotation (Line(points={{-60,-20},{-50,-20},{-50,-10}}, color={0,0,255}));
+  connect(capacitorLV.n, dcdc.dc_n1) annotation (Line(points={{-50,-10},{-50,-20},
+          {0,-20},{0,-4},{10,-4}},   color={0,0,255}));
+  annotation (experiment(
+      StopTime=1,
+      Interval=1e-05,
+      Tolerance=1e-06), Documentation(info="<html>
+<p>
+Buck/boost chopper example template including both voltage sources; control not included yet
```

Comment:
What do you mean by "control not included yet"? Is it planned to be added to this model in the near future? Otherwise, I'd remove the "yet", which gives an impression of work-in-progress

Reply:
You're right, I plan to implement control, but I don't have enough time to include it in the near future.